app/assets/javascripts/controllers/DragDropCtrl.coffee

```diff
@@ -0,0 +1,85 @@
+# TODO: create modal for save option?
+@bruseApp.controller 'DragDropCtrl', ['$scope', '$http', ($scope, $http) ->
+    $scope.fileObject = null
+    $scope.dataObject = {location:''}
+    $scope.imageFile = null
+    $scope.isDropped = false
+    $scope.isSaved = false
```

Comment:
In my opinion the initiation of `false` and `null` variables is unnecessary because they aren't used in this file. Since `undefined` is interpreted as `false` all validations will work as before.